README.md

```diff
@@ -68,9 +68,10 @@ let (vk_bytes, sig_bytes) = {
 };
 
 // Verify the signature
+
 assert!(
     VerificationKey::try_from(vk_bytes)
-        .and_then(|vk| vk.verify(&sig_bytes.into(), msg))
+        .and_then(|vk| vk.verify(&sig_bytes.try_into().unwrap(), msg))
```

Comment:
`ed25519::Signature` does a check that the 3 highest bits of `s` are 0, versus the infallible constructor `ed25519_consensus::Signature` provides currently. If there are strong feelings about that it can be changed to be infallible in the `ed25519` crate as well as we're currently in a phase of breaking changes prior to a 2.0 release.

If the fallible constructor is retained, this example could probably be better, e.g. `let sig = Signature::try_from(&sig_bytes)?;`

Reply:
My preference would be to keep the conversion infallible. At least for Ed25519, I'm not sure how to understand a meaningful difference between "invalid signature because invalid R or s values" vs "invalid signature because 3 highest bits of s are invalid", and in `ed25519_consensus` we want to do a full check that `s` is canonically encoded.  So it seems better to keep all of the verification logic in one place and have an up-or-down answer.

[I would make a distinction here between signature data and verification key data; having a `VerificationKey` that precomputes some checks for the verification key seems useful because you may reuse it multiple times, while signatures are generally only checked once.]

Reply:
This check has been removed and the conversion is now infallible.